app/jobs/usage_monitoring/process_organization_subscription_activities_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module UsageMonitoring
+  class ProcessOrganizationSubscriptionActivitiesJob < ApplicationJob
+    unique :until_executed, on_conflict: :log
+
+    def perform(organization_id)
+      return unless License.premium?
+
+      organization = Organization.find(organization_id)
+      result = UsageMonitoring::ProcessOrganizationSubscriptionActivitiesService.call!(organization:)
+
+      Rails.logger.info(
```

Comment:
Not a blocker, but since job enqueuing is always logged anyway, do we really need this one?